Makefile

```diff
@@ -88,11 +62,7 @@ lint: gopath
 	./...
 
 clean:
-ifeq (,${LOCAL_GOPATH})
-	go clean -i -x
-else
-	rm -rf ${LOCAL_GOPATH}
-endif
+	go clean -i -x ./...
```

Comment:
Just curious about why you need `./...` here

Reply:
Because without it `go clean` errors out with
```
can't load package: package github.com/clearlinux/mixer-tools: unknown import path "github.com/clearlinux/mixer-tools": cannot find module providing package github.com/clearlinux/mixer-tools
```
since mixer-tools itself doesn't provide a go package (its subdirectories do)